contentful/apps
# refactors fetch into a hook

## Purpose
Makes it easier to work in/move logic around

## Approach
Hooks make it easier to reuse logic and pull out of a lot of the logic 

## Testing steps

- Go to https://app.contentful.com/spaces/juh8bvgveao4/entries/7myomzlWGB8F47CWhyvAB6
- Verify loading state happens and resolves
